modules/usbInterface/usbInterface.py

```diff
@@ -0,0 +1,36 @@
+import usb.core
+import usb.util
+
+class USBInterface:
```

Comment:
This is pretty perfect, I'm just going to ask for one extension. Firmware told me a day or so ago they want to do UART as well, so we'll implement both. Here's how I want to do it.

First, define a base class CommsInterface, with the same methods as USB Interface. Make them all abstract methods. Make USBInterface a subclass of CommsInterface
Then, implement the same logic for UART, creating endpoints, read and write according to the [PySerial Docs](https://pyserial.readthedocs.io/en/latest/pyserial.html#overview) and [the confluence doc](https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/1952022529/STM32-Jetson+Communication+Architecture)